games/achievements.py

```diff
@@ -11,13 +12,25 @@
 TIMEZONE_FILENAME = zoneinfo._tzpath.find_tzfile("Europe/Copenhagen")
 with open(TIMEZONE_FILENAME, "rb") as f:
     TIMEZONE_DATA = zoneinfo._common.load_data(f)
-
+VALID_DATES = filter(lambda t: t > 0, TIMEZONE_DATA[1])
 DST_TRANSITION_TIMES = [
-    datetime.datetime.fromtimestamp(t, tz=datetime.timezone.utc)
-    for t in TIMEZONE_DATA[1]
+    datetime.datetime.fromtimestamp(t, tz=datetime.timezone.utc) for t in VALID_DATES
 ]
 
 
+class AchievementLevel(IntEnum):
+    """Each level must correspond to a style in styles.css"""
+
+    GOLD = 4
+    SILVER = 3
+    BRONZE = 2
+    BASE = 1
+    NO_LEVEL = 0
```

Comment:
Isn't the css classes broken now? Doesn't this add the class `0`?

Reply:
I would also rather keep it as a strEnum. With enums, one can get a number by inspecting the ._member_names_ attribute, which will always be in definition order.

Ultimately something like

```
class AchievementLevel(IntEnum):
    """
    Each level must correspond to a style in styles.css, 
    and be defined in increasing order of difficulty.
    """

    NO_LEVEL = "no_level"
    BASE = "base"
    BRONZE = "bronze"
    SILVER = "silver"
    GOLD = "gold"

    def __lt__(self, other):
        return self._member_names_.index(self) < self._member_names_.index(other)
```

Reply:
I was not aware you could get the index like that; I would agree on that being the better approach